protocols/netlink/netlink.go

```diff
@@ -0,0 +1,45 @@
+package protocolnetlink
+
+import (
+	"github.com/bio-routing/bio-rd/config"
+	"github.com/bio-routing/bio-rd/routingtable"
+	"github.com/bio-routing/bio-rd/routingtable/locRIB"
+)
+
+// Netlink is the netlink module which handles the entire NetlinkCommunication
+type Netlink struct {
+	locRib *locRIB.LocRIB
+
+	writer *NetlinkWriter
+	reader *NetlinkReader
+}
+
+// NewNetlink creates a new Netlink object and returns the pointer to it
+func NewNetlink(options *config.Netlink, locRib *locRIB.LocRIB) *Netlink {
+
+	n := &Netlink{
+		locRib: locRib,
+		writer: NewNetlinkWriter(options),
+		reader: NewNetlinkReader(options),
+	}
+	return n
+}
+
+// Start the Netlink module
+func (n *Netlink) Start() {
+	// connect all RIBs
+	options := routingtable.ClientOptions{
```

Comment:
I guess we only want to push the ECMP routes into the kernel. Not all we have.

Reply:
I would change that within the subject of the ECMP for Netlink PR (is coming very soon, already started working on it)

Reply:
as discussed: this is addressed in #133